### PR TITLE
Billy/762 bech32

### PR DESCRIPTION
**Pull request by okwme**
Closes #ISSUE #762 

Description:
make all addresses bech32 compliant and checksummed

### CODE REVIEW

**Comment by okwme** [comment]
pinging @fabo or @mappum the e2e tests fail because the users are generated via lcd command line and their addresses are stored in non-bech32 format—therefor rejected when attempting to store them in the state. I wasn't sure where I could intervene to transform them or rather that should happen from the beginning and we just need to wait for the new sdk release?

**Comment by faboweb** [comment]
If the bech32 change is in the SDK and the testnet supports that version we can upgrade and use it in Voyager. 
For now the change is merged to develop on the SDK so we could test with a custom build.

**Comment by faboweb** [comment]
I found some bugs in my bech32 implementation in the SDK. I need to fix those first.

**Comment by faboweb** [comment]
This is now mostly working. @okwme could you fix the e2e tests?

**Comment by okwme** [comment]
think the tests are working except those that fail because the testnet

**Comment by faboweb** [comment]
After intense debugging we found that probably the signing in the LCD is broken. We will fix this now.

**Comment by okwme** [comment]
so the error message says `encoding/hex: invalid byte: U+006F 'o'` when sending to address `cosmosaccaddr1pxdf0lvq5jvl9uxznklgc5gxuwzpdy5ynem545`

File: app/src/renderer/connectors/lcdClientMock.js [hunk 1]
```diff
@@ -375,7 +385,11 @@ module.exports = {
   },
   async candidates() {
     return state.candidates
-  }
+  },
+  // exports to be used in tests
```
Comment (okwme) [inline app/src/renderer/connectors/lcdClientMock.js]: nice

**Pull request merged**